.gitignore

```diff
@@ -1,4 +1,6 @@
 .psci
 .psci_modules
+.history
```

Comment:
To be honest I don't like this kind of global settings, since it can introduce bugs in the future.
The problem is that either this directory should be excluded or not. 
But I understand your point of view. I will remove this line